components/TextFields/examples/MultilineTextFieldLegacyExample.m

```diff
@@ -158,6 +157,61 @@ - (void)viewDidLoad {
                                                       @"charMax" : multilineTextFieldCharMaxDefault
                                                     }]];
 
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    [NSLayoutConstraint activateConstraints:@[
+      [NSLayoutConstraint constraintWithItem:multilineTextFieldUnstyled
+                                   attribute:NSLayoutAttributeTop
+                                   relatedBy:NSLayoutRelationEqual
+                                      toItem:self.scrollView.contentLayoutGuide
+                                   attribute:NSLayoutAttributeTop
+                                  multiplier:1
+                                    constant:20],
+      [NSLayoutConstraint constraintWithItem:multilineTextFieldCharMaxFullWidth
+                                   attribute:NSLayoutAttributeBottom
+                                   relatedBy:NSLayoutRelationEqual
+                                      toItem:self.scrollView.contentLayoutGuide
+                                   attribute:NSLayoutAttributeBottomMargin
+                                  multiplier:1
+                                    constant:-20]
+    ]];
+  } else {
+    [NSLayoutConstraint activateConstraints:@[
```

Comment:
The two "elses" look the same to me and I see this is a pattern repeated throughout these examples. I would prefer to see either:
1. Make a function/method that handles the "else" case so we don't have 2 updates in the future
2. Make it a single if/else with macros protecting the else?
```objectivec
#if __IPHONE_11
  if (@available) {
    ... do iOS 11 stuff...
  }
  else {
#endif
  ... do iOS 8-10 stuff....
#if __IPHONE_11
  } // @available else
#endif
```

Reply:
Well, here's the thing, aside from kitchen sink, this is all educational code so I don't want to get too clever. This is the minimum amount of fancy footwork needed to make it all work.

Reply:
But this is what I mentioned in my email. We all need to agree on the approach. Put your suggestion in that thread too.

Reply:
Ok. I put it in the thread, and am OK with this as-is. Thanks!

Reply:
What I did in a similar situation was to use a separate method for setting up the constraints and return if we do have the iOS 11 API available:
https://github.com/material-components/material-components-ios/pull/2021/files#diff-3f90132dc1e2e304c7961cb3e9d58a40R304

Reply:
Return early. That's an unexpected twist. I think for examples, it would be too easy to miss.